src/bundlers/node/containerManifest.ts

```diff
@@ -0,0 +1,30 @@
+export const node16image = "node:16.20.2-alpine3.18";
+export const node18image = "node:18.19.0-alpine";
```

Comment:
Since v1.0.0 - we are building with `node20` by default if the user doesn't specify this.

I assume that this file should be updated also to reflect that.

Reply:
Also, probably you can remove the hardcoded values and use the array defined here [1] to do a switch case.
It will also help to add/deprecated new node versions in the future.

Feel free to add new global values such as `"node:16.20.2-alpine3.18"` or `"node:18.19.0-alpine";`.

[1] https://github.com/Genez-io/genezio/blob/main/src/models/nodeRuntime.ts#L6